cloud-native-runtimes/install-cnrt.hbs.md

```diff
@@ -65,41 +65,34 @@ To install Cloud Native Runtimes:
 
         ```console
         ---
-        # if deploying on a local cluster such as Kind. Otherwise, you can remove this field
-        provider: local
+        domain_name: example.com
+        ingress:
+        external:
+            namespace: tanzu-system-ingress
+        internal:
+            namespace: tanzu-system-ingress
         ```
 
         >**Note:** For most installations, you can leave the `cnr-values.yaml` empty, and use the default values.
 
-        If you are running on a single-node cluster, such as kind or minikube, set the `provider: local`
-        option. This option reduces resource requirements by using a HostPort service instead of a
-        LoadBalancer and reduces the number of replicas.
-
-        Cloud Native Runtimes reuses the existing `tanzu-system-ingress` Contour installation for
-        external and internal access when installed in the `light` or `full` profile.
-        If you want to use a separate Contour installation for system-internal traffic, set
-        `cnrs.ingress.internal.namespace` to the empty string (`""`).
-
-        For more information about using Cloud Native Runtimes with kind, see the
-        [Cloud Native Runtimes documentation](https://docs.vmware.com/en/Cloud-Native-Runtimes-for-VMware-Tanzu/1.2/tanzu-cloud-native-runtimes/GUID-local-dns.html#config-cluster).
+        If you are running on a single-node cluster, such as kind or minikube, set the `lite.enable: true`
+        option. This option reduces resources requests for CNR deployments.
         If you are running on a multinode cluster, do not set `provider`.
```

Comment:
We need to update provider -> lite here as well.

Other than that, LGTM!

Reply:
okay I wasn't sure what you were referring to here, so I just dropped the line about running on a multinode cluster. I don't think it made sense anyway